src/command.ts

```diff
@@ -92,6 +94,8 @@ export default abstract class Command {
 
   static disableJsonFlag: boolean | undefined
 
+  static errorCodes?: ErrorCode[];
```

Comment:
Add new static property for command to hold user defined error codes

Reply:
The more I think about it, the more I think this should belong in our custom help class instead of oclif

My main concern is not overwhelming oclif users with all the possible options

Reply:
I will move error codes elsewhere